6.0.0-beta.14 Vue SFC scss @import trying to resolve url() even tho processCssUrls is set to false

-   Laravel Mix Version: 6.0.0-beta.14
-   Node Version (`node -v`): v15.2.0
-   NPM Version (`npm -v`): 7.0.8
-   OS: Ubuntu 18.04 on Windows 10

### Description:
in my `webpack.mix.js` i've set `processCssUrls: false` however when i run `mix.js(/**/).vue()` one of my Single File Vue Components (SFC) which has a `scss` `@import` statement, imports a stylesheet that has a `url()` background-image and a font src:url() definition. Compilation fails with "Can't resolve" errors.

Is there some special extra config option i need to send into Mix or Webpack to get vue-loader to not processCssUrls when compiling styles?

### Steps To Reproduce:
webpack@5.6.0
vue@2.6.12
vue-loader15 (i assume)
sass@1.29

```
 ERROR  Failed to compile with 2 errors                                                                                                                                                                  4:01:01 PM

 error  in ./components/Component.vue?vue&type=style&index=0&id=4b38ac70&lang=scss&scoped=true&

Syntax Error: Error: Can't resolve '/fonts/vjs.eot' in 'components'


 error  in ./components/Component.vue?vue&type=style&index=0&id=860b7dd0&lang=scss&scoped=true&

Syntax Error: Error: Can't resolve '/images/subnav_mobile.jpg' in 'components'



npm ERR! code 1
npm ERR! path /root/app
npm ERR! command failed
npm ERR! command sh -c NODE_ENV=development webpack --progress --config=node_modules/laravel-mix/setup/webpack.config.js "--env" "js-only"

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-25T00_01_01_436Z-debug.log
```


**UPDATE** i tried adding `extractVueStyles: true` but that didn't seem to make a difference

**UPDATE 2**
```
if i add this to `webpackConfig.module.rules[]` it compiles:
{
        test: /\.vue$/,
        loader: 'vue-loader',
        options: {
          loaders: {
            scss: 'style-loader!css-loader?url=false!sass-loader'
          }
        }
      }
```

**UPDATE THE THIRD** ^ that had bad side-effects elsewhere, so now i'm trying the slightly _more_ generalized:
```
mix.override((webpackConfig)=>{
  // console.log(webpackConfig.module.rules);
  webpackConfig.module.rules = _.map(webpackConfig.module.rules,function(rule){
    // console.log(rule);
    if(rule.test == "/\\.scss$/"){
      rule.oneOf = _.map(rule.oneOf, function(oneOf){
        oneOf.use = _.map(oneOf.use, function(use){
          if(use.loader === "css-loader"){
            use.options.url = false;
          }
          return use;
        })
        return oneOf
      })
    }
    return rule
  })
})
```

but that still feels gross, and like something mix should provide an easy escape hatch for

--- a/src/components/CssWebpackConfig.js
+++ b/src/components/CssWebpackConfig.js
@@ -104,7 +104,12 @@ class CssWebpackConfig extends AutomaticComponent {
     createLoaderList(rule, useCssModules) {
         return [
             ...CssWebpackConfig.afterLoaders(),
-            { loader: 'css-loader', options: { modules: useCssModules } },
+            {
+                loader: 'css-loader',
+                options: {
+                    modules: useCssModules
+                }
+            },
             {
                 loader: 'postcss-loader',
                 options: {

--- a/src/components/CssWebpackConfig.js
+++ b/src/components/CssWebpackConfig.js
@@ -107,6 +107,7 @@ class CssWebpackConfig extends AutomaticComponent {
             {
                 loader: 'css-loader',
                 options: {
+                    url: Config.processCssUrls,
                     modules: useCssModules
                 }
             },

--- a/test/features/vue.js
+++ b/test/features/vue.js
@@ -7,7 +7,9 @@ import assert from '../helpers/assertions';
 
 import '../helpers/mix';
 
-test.beforeEach(() => webpack.setupVueAliases(2));
+test.beforeEach(() => {
+    webpack.setupVueAliases(2);
+});
 
 test('it adds the Vue 2 resolve alias', async t => {
     mix.vue({ version: 2, extractStyles: true });

--- a/test/features/vue.js
+++ b/test/features/vue.js
@@ -9,6 +9,7 @@ import '../helpers/mix';
 
 test.beforeEach(() => {
     webpack.setupVueAliases(2);
+    mix.options({ processCssUrls: false });
 });
 
 test('it adds the Vue 2 resolve alias', async t => {
@@ -56,6 +57,7 @@ test('it appends vue styles to your sass compiled file', async t => {
 
 .hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 
@@ -80,6 +82,7 @@ test('it appends vue styles to your less compiled file', async t => {
 
 .hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 
@@ -96,6 +99,7 @@ test('it appends vue styles to a vue-styles.css file, if no preprocessor is used
 
     let expected = `.hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 
@@ -172,6 +176,7 @@ test('it extracts vue .scss styles to a dedicated file', async t => {
 
     expected = `.hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 

--- a/test/features/vue3.js
+++ b/test/features/vue3.js
@@ -8,6 +8,7 @@ import '../helpers/mix';
 
 test.beforeEach(() => {
     webpack.setupVueAliases(3);
+    mix.options({ processCssUrls: false });
 });
 
 test('it adds the Vue 3 resolve alias', async t => {
@@ -53,6 +54,7 @@ test('it appends vue styles to your sass compiled file', async t => {
 
 .hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 
@@ -77,6 +79,7 @@ test('it appends vue styles to your less compiled file', async t => {
 
 .hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 
@@ -94,6 +97,7 @@ test('it appends vue styles to a vue-styles.css file, if no preprocessor is used
 
     let expected = `.hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 
@@ -168,6 +172,7 @@ test('it extracts vue .scss styles to a dedicated file', async t => {
 
     expected = `.hello {
   color: blue;
+  background: url(./does-not-exist.png);
 }
 `;
 

--- a/test/fixtures/app/src/vue/BasicWithScss.vue
+++ b/test/fixtures/app/src/vue/BasicWithScss.vue
@@ -15,5 +15,6 @@ module.exports = {
 <style lang="scss">
 .hello {
     color: blue;
+    background: url(./does-not-exist.png);
 }
 </style>

--- a/test/fixtures/app/src/vue3/BasicWithScss.vue
+++ b/test/fixtures/app/src/vue3/BasicWithScss.vue
@@ -15,5 +15,6 @@ module.exports = {
 <style lang="scss">
 .hello {
     color: blue;
+    background: url(./does-not-exist.png);
 }
 </style>

